Kickstarter-iOS/Views/Controllers/LoginViewController.swift

```diff
@@ -166,6 +174,14 @@ internal final class LoginViewController: UIViewController {
     self.navigationController?.pushViewController(vc, animated: true)
   }
 
+  fileprivate func startEmailVerificationViewController() {
+    /**
+     FIXME: `UIViewController` needs to be replaced with the EmailVerification UI when development is complete.
```

Comment:
Is leaving this comment in until the view controller UI is completed OK or is there another preferred way to handle it?